### PR TITLE
Hawkish/dovish trajectory model + panel (#296)

### PR DESCRIPTION
**Pull request by yusufizzetmurat**
## Summary

- LSTM baseline + Transformer arm under `app/trajectory/`; same input contract
- Walk-forward trainer with `--train-end` / `--fold-id`; atomic bundle writes
- `/analyze/trajectory` route + `app.services.trajectory` singleton with sanitized 503
- `TrajectoryPanel` scatter chart with projected-next star + confidence band
- Cross-bank DAPT encoder embeddings + #291 pre-meeting market features
- APS conformal calibration on next-meeting softmax

## Test plan

- [ ] `docker compose -p fed-pulse-bundled run --rm backend pytest tests/unit/test_trajectory_model.py tests/unit/test_trajectory_train.py tests/unit/test_trajectory_endpoint.py -q`
- [ ] `docker compose -p fed-pulse-bundled run --rm backend mypy --strict --ignore-missing-imports --follow-imports=silent app/trajectory app/services/trajectory.py`
- [ ] `cd frontend && npm run build`

### CODE REVIEW

**Comment by yusufizzetmurat** [comment]
Review fixes pushed (commit 3c6ddb8):

- (1) _slice_history strict-backward (< not <=)
- (2) PCA fit on TRAIN-slice; pca_mean/components persisted
- (3) standardise_inputs fit on TRAIN ONLY (split-first, then fit)
- (4) Holdout = post-train_end to fold test_end (cal = tail of train)
- (5) persist_bundle filters event_date < train_end; warning when as_of beyond train_end
- (6) parquet carries pre_meeting_trailing_2y_yield_change_5d_bps + vix_close
- (7) Transformer all-pad guard prevents softmax(-inf) NaN
- (8) market vec marks missing via bias bit; NaN-safe; no > 0 gating
- (9) NpzFile probe via .files (no .get KeyError swallow)
- (10) text_hash lookup disambiguates duplicate event_date rows
- (11) Double-checked locking on get_state (load outside lock)
- (12) torch.load(weights_only=True) + JSON config payload
- (13) model_parameter_count in manifest + metrics; parity-ratio test
- (14) all-pad + masked tests parametrised over LSTM + Transformer
- (15) Atomic-crash test asserts no manifest survives a mid-write fail
- (16) Real-encoder end-to-end test (slow marker; skips offline)
- (17) Conformal cal-slice spy confirms calibration partition is fed

Tests: 45 passed, 1 skipped (slow). Lint + mypy strict clean. Frontend builds clean.
